### PR TITLE
refactor(grid-header): [grid]add TInyGridHeader renderless logic

### PR DESCRIPTION
**Pull request by AcWrong02**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: packages/renderless/types/grid/header.type.ts
Comment (Zcating) [inline packages/renderless/types/grid/header.type.ts:1]: 不要新建文件夹，打平，grid-header.type.ts

**Review by Zcating — CHANGES_REQUESTED** [review]

File: packages/vue/src/grid/src/header/src/pc.vue [hunk 1]
```diff
@@ -340,25 +309,22 @@ function renderTableThead(args) {
...
-
 function renderTable(args) {
+  // 将传入的args再解构出来
   let { $table, _vm, allAlign, allColumnHeaderOverflow, allHeaderAlign, border, columnKey } = args
   let { headerCellClassName, headerColumn, headerRowClassName, headerSuffixIconAbsolute } = args
   let { highlightCurrentColumn, isDragHeaderSorting, mouseConfig, overflowX, resizable, sortOpts } = args
   let { tableColumn, tableLayout, tableListeners } = args
+
+  // 定义另一个args1
+  // Q：为什么要再定义出一个？有什么区别吗？
```
Comment (gimmyhehe) [inline packages/vue/src/grid/src/header/src/pc.vue:327]: 其实没有区别，我已经将部分这种用法改成了传递gridVm，然后在具体函数再解构

File: packages/renderless/types/grid/header.type.ts
Comment (AcWrong02) [reply packages/renderless/types/grid/header.type.ts:1]: get

**Review by Zcating — APPROVED** [review]

**Comment by kagol** [comment]
@AcWrong02 麻烦解决下冲突~
image

**Pull request closed without merging**